Data/sdLitica.Relational/Repositories/AnalyticsModuleRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.EntityFrameworkCore;
+using sdLitica.Entities.Analytics;
+using sdLitica.Relational.Context;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace sdLitica.Relational.Repositories
+{
+    /// <summary>
+    /// This class provides data access operations for AnalyticsModule
+    /// </summary>
+    public class AnalyticsModuleRepository : RepositoryBase<AnalyticsModule>
+    {
+        public AnalyticsModuleRepository(MySqlDbContext context)
+            : base(context)
+        {
+        }
+
+        /// <summary>
+        /// Check whether element given by guid exists
+        /// </summary>
+        /// <param name="guid"></param>
+        /// <returns></returns>
+        public bool Contains(Guid guid)
+        {
+            return Entity.Any(p => p.Id.Equals(guid));
+        }
+
+        /// <summary>
+        /// Get all modules with related operations.
+        /// </summary>
+        /// <returns></returns>
+        public IList<AnalyticsModule> GetAll()
```

Comment:
@rsbavaresco this is a good question.

I will try to start from some assomptions. First is the size of this list. How many different modules may we have in nearest future? 10? 50? 100? I don't think it will be meaningfully more then 100. So, for the list of 100 items paginations may be not profitable enough comparing to pagination implementation complexity. What do you think?

About point 3 I will agree in general, but here table looks relatively small.